plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -441,6 +481,56 @@ public void setNull(PreparedStatement statement, int index)
         };
     }
 
+    public static ColumnMapping longTimestampColumnMapping(TimestampType type)
```

Comment:
The method is named as-if it was returning mapping for a "long timestamp", but it works for both short & long.

`timestampColumnMapping`

Reply:
I import `timestampColumnMapping`, that's actually only works for short timestamps. How about `anyTimestampColumnMapping`?

Reply:
`sqlServerTimestampColumnMapping`